src/app/modules/document-search/document-search.component.spec.ts

```diff
@@ -225,17 +244,23 @@ describe('DocumentSearchComponent', () => {
     component.setFilterActions();
     expect(component.items).toBeTruthy();
     expect(component.items.length).toBe(3);
-    expect(component.items[0].label).toBe(undefined);
+    expect(component.items[0].label).toBe(
+      'DOCUMENT_SEARCH.FILTER.CREATED_BY_ME'
+    );
     expect(component.items[0].command).toBeDefined();
     spyOn(component, 'getFilteredCreatedByMe');
     component.items[0].command();
     expect(component.getFilteredCreatedByMe).toHaveBeenCalled();
-    expect(component.items[1].label).toBe(undefined);
+    expect(component.items[1].label).toBe(
+      'DOCUMENT_SEARCH.FILTER.RECENTLY_UPDATED'
+    );
     expect(component.items[1].command).toBeDefined();
     spyOn(component, 'getFilteredRecentlyUpdated');
     component.items[1].command();
     expect(component.getFilteredRecentlyUpdated).toHaveBeenCalled();
-    expect(component.items[2].label).toBe(undefined);
+    expect(component.items[2].label).toBe(
+      'DOCUMENT_SEARCH.FILTER.CLEAR_FILTER'
```

Comment:
Why has this changed?

Reply:
I reverted the changes, Previously! the testcases are failing due to translation issue.